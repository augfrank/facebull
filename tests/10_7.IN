M0   C0   C1  2980
M1   C0   C2  2877
M2   C0   C3  2599
M3   C0   C4  2583
M4   C0   C5  2148
M5   C0   C6  2228
M6   C0   C8  2618
M7   C0   C9  2022
M8   C1   C2  2285
M9   C1   C3  2635
M10   C1   C4  2832
M11   C1   C5  2599
M12   C1   C6  2196
M13   C1   C7  2444
M14   C1   C8  2563
M15   C2   C0  2627
M16   C2   C1  2661
M17   C2   C3  2249
M18   C2   C4  2593
M19   C2   C6  2129
M20   C2   C7  2235
M21   C2   C8  2998
M22   C3   C0  2913
M23   C3   C1  2793
M24   C3   C5  2043
M25   C3   C7  2589
M26   C3   C9  2232
M27   C4   C1  2423
M28   C4   C2  2953
M29   C4   C5  2477
M30   C4   C6  2408
M31   C4   C7  2718
M32   C4   C9  2655
M33   C5   C0  2476
M34   C5   C1  2054
M35   C5   C2  2398
M36   C5   C4  2996
M37   C5   C6  2109
M38   C5   C7  2064
M39   C5   C8  2046
M40   C5   C9  2224
M41   C6   C0  2571
M42   C6   C1  2529
M43   C6   C2  2020
M44   C6   C3  2748
M45   C6   C4  2188
M46   C6   C5  2179
M47   C6   C8  2906
M48   C7   C0  2878
M49   C7   C1  2516
M50   C7   C2  2628
M51   C7   C3  2367
M52   C7   C4  2074
M53   C7   C5  2005
M54   C7   C6  2599
M55   C7   C8  2040
M56   C7   C9  2903
M57   C8   C0  2141
M58   C8   C1  2477
M59   C8   C2  2789
M60   C8   C3  2599
M61   C8   C4  2432
M62   C8   C5  2271
M63   C8   C9  2461
M64   C9   C0  2905
M65   C9   C1  2501
M66   C9   C2  2166
M67   C9   C3  2109
M68   C9   C4  2354
M69   C9   C5  2584
M70   C9   C6  2503
M71   C9   C7  2374
