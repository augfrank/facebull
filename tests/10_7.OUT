21552
7 12 17 24 27 38 43 55 57 68
